examples/platform/silabs/BaseApplication.cpp

```diff
@@ -505,13 +506,9 @@ void BaseApplication::ButtonHandler(AppEvent * aEvent)
             else
             {
                 SILABS_LOG("Network is already provisioned, Ble advertissement not enabled");
-                DeviceLayer::ChipDeviceEvent event;
-                event.Type     = DeviceLayer::DeviceEventType::kAppWakeUpEvent;
-                CHIP_ERROR err = DeviceLayer::PlatformMgr().PostEvent(&event);
-                if (err != CHIP_NO_ERROR)
-                {
-                    ChipLogError(AppServer, "Failed to post App wake up Event event %" CHIP_ERROR_FORMAT, err.Format());
-                }
+#if CHIP_CONFIG_ENABLE_ICD_SERVER
+                GetICDNotifier()->NetworkActivity();
```

Comment:
This will become something else when it is well defined in the spec. similar to the `StayActiveRequest Command` Currently it only triggers 1 active Threshold so it is the same functionality than NetWorkActivity